hardware/Comply-to-the-Regulations.md

```diff
@@ -0,0 +1,394 @@
+  - Highlights the modular approval type
+- Test Reports 
+  - Zip packages for region or authority containing various PDF reports for customer reference
+  - Find these in Technical Resource Search under miscellaneous category
+- Installation Guide
+  - Detailed layout guidelines, including dimensions
+  - Possibly includes FCC “test list” help
+  - Possibly includes other general regulatory advice
+- Layout Template 
+  - File that is importable into a CAD design on a non-printable layer that shows outlines and dimensions that would help OEMs
+  - It would also greatly aid in Apps reviews of customer designs
+- Design Files for Modules: Module design files are Silicon Labs IP so cannot be shared with customers in general. But if it is required for certification purposes, Silicon Labs can share the design files with the certification test house under NDA in-placed with. 
+
+## 5.2. Process flow in certifications
+- For FCC:
+  1. Prepare samples and book a test slot from an accredited test lab
+  2. Test lab will test according to the test instructions given by the module manufacturer
+  3. Test lab provides the test reports
+  4. Module manufacturer submits test reports and a certification application to a Certification Body
+  5. The Certification Body reviews the application and admits the certificate to the manufacturer
+  6. The CB uploads the certification listing to the specific database
+- For CE:
+  1. Prepare samples and book a test slot from an accredited test lab
+  2. Test lab will test according to the test instructions given by the module manufacturer
+  3. Test lab provides the test reports
+  4. a. If all testing was done according to harmonized standards
+     - Manufacture files the test evidence and all technical material to Technical Construction File (TCF).
+     - Manufacturer will sign a Declaration of Conformity in which the manufacturer states all the standards that the product is found to be compliant with.
+
+    b. If all testing was not done according to harmonized standards
+     - Manufacturer submits a NB application to a NB (Notified Body) 
+     - The NB will review the test reports and all technical material and user manuals
+     - If the NB approves, he will provide the manufacturer with a letter in which he states that he has found the product to be compliant with all applicable standards
+     - Manufacture files the NB letter, test evidence and all technical material to Technical Construction File (TCF).
+     - Manufacturer will sign a Declaration of Conformity in which the manufacturer states all the standards that the product is found to be compliant with.
+
+## 5.3. Engineering flow
+The following flow-chart is designed for customer to certificate its design that can pass emissions requirements. It is a typical engineer flow for whatever the regulation region cared about. Customer will need to design its own system and test it according to this flow even when using a fully certified module.
```

Comment:
Change "certificate" to "certify".  Change "engineer" to "engineering"

Reply:
Corrected.